packages/theme/css/cssVariables.json

```diff
@@ -0,0 +1,11 @@
+{
+  "position": "relative",
+  "right": "0",
+  "top": "50%",
+  "transform": "translateY(-50%)",
+  "width": "unset",
+  "display": "none",
+  "margin-bottom": "5%",
```

Comment:
could we use spacing tokens?

Reply:
this file was a test generation so i deleted it